app/views/v1/actions/index.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.count @activity.actions.count
```

Comment:
I assume the count should be total count of actions, not actions that appear in results?